.gitignore

```diff
@@ -7,4 +7,5 @@
 /pkg/
 /spec/reports/
 /tmp/
+/bin/
```

Comment:
I was following [the RSpec install recommendation](http://rspec.info/) which created that directory. I didn’t want to bundle it with the repo.